src/App.js

```diff
@@ -1,18 +1,22 @@
 import React, { useEffect, useState } from "react";
-import cs from "classnames";
+
 import "./App.css";
 import Foe from "./Foe/Foe";
 import Player from "./Player/Player";
+import Menu from "./Menu/Menu";
 
 export default function App() {
-    const [isMenuHidden, setIsMenuHidden] = useState(false);
-    const [isBattleMenuHidden, setIsBattleMenuHidden] = useState(true);
-    const [isPartyMenuHidden, setIsPartyMenuHidden] = useState(true);
-    const [isItemMenuHidden, setIsItemMenuHidden] = useState(true);
     const [playerHP, setPlayerHP] = useState();
```

Comment:
I have been thinking about whether or not this and the opponentHP should be added as part of the respective pokemon objects. On one hand it will clean up these variables and make  them part of the objects (less stuff to pass as props) but on the other hand you have to write slightly more when you actually do the health calculations.

So I'll leave this one up to you. Make the choice you think works best for you